models/models.js

```diff
@@ -0,0 +1,47 @@
+const client = require('../DB/db.js');
+
+const runQuery = (query, callback) => {
```

Comment:
What is this function doing? I can't find it used anywhere, so i'm thinking it's probably just a template